### PR TITLE
Fix lints issue.

### PR DESCRIPTION
**Pull request by pzheng1025**

### CODE REVIEW

File: src/fuse/ll_request.rs [hunk 1]
```diff
@@ -7,7 +7,15 @@ use std::convert::TryFrom;
 use std::ffi::OsStr;
 use std::{error, fmt, mem};
 
-use super::abi::*;
+#[cfg(target_os = "macos")]
+use super::abi::fuse_exchange_in;
+use super::abi::{
```
Comment (MrWWheat) [inline src/fuse/ll_request.rs:12]: Can not merge?

File: src/fuse/mount.rs [hunk 2]
```diff
@@ -8,7 +8,14 @@ use std::fs;
 use std::os::unix::io::RawFd;
 use std::path::Path;
 
-use param::*;
+#[cfg(target_os = "macos")]
+use param::{
+    copy_slice, parse_mount_flag, FUSE_IOC_MAGIC, FUSE_IOC_TYPE_MODE, MAXPATHLEN, MNT_NOATIME,
+    MNT_NODEV, MNT_NOSUID, MNT_NOUSERXATTR,
+};
+use param::{get_mount_options, FuseMountArgs, MNT_FORCE};
```
Comment (MrWWheat) [inline src/fuse/mount.rs:16]: Should we merge them?

File: src/fuse/ll_request.rs (hunk 1 above)
Comment (pzheng1025) [reply src/fuse/ll_request.rs:12]: use super::abi::fuse_exchange_in;  This is only needed by macos.

File: src/fuse/mount.rs (hunk 2 above)
Comment (pzheng1025) [reply src/fuse/mount.rs:16]: The same as last one.

**Pull request merged**